deep-learning-tools/templates/ingress.yaml

```diff
@@ -10,6 +10,8 @@ metadata:
   annotations:
     kubernetes.io/ingress.class: nginx
     kubernetes.io/tls-acme-staging: "true"
+    nginx.ingress.kubernetes.io/proxy-send-timeout: 7200
```

Comment:
The timeout seemingly needs to be quoted as a string (i.e. "7200" instead of 7200), as a string is apparently expected.
Otherwise this works nicely :)